src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -498,7 +504,23 @@ def shim_to_feedback(
             },
         }
 
-        feedback_event["contexts"]["feedback"]["associated_event_id"] = event.event_id
+        # An invalid event_id leads to tracking the outcome and dropping the feedback
+        try:
+            UUID(event.event_id, version=4)
+            feedback_event["contexts"]["feedback"]["associated_event_id"] = event.event_id
+        except ValueError:
+            track_outcome(
+                org_id=project.organization_id,
+                project_id=project.id,
+                key_id=None,
+                outcome=Outcome.RATE_LIMITED,
```

Comment:
I'd expect a different outcome here, otherwise can't debug the actual reason.
Is there an existent outcome that can do the job? Not sure about the process of creating new ones

```suggestion
                outcome=Outcome.XYZ,
```

Reply:
This was an oversight, Outcome.INVALID seems to make more sense here

Reply:
INVALID is prob best here

Reply:
Beat me to it!